manual/cops_bundler.md

```diff
@@ -6,11 +6,36 @@ Enabled by default | Supports autocorrection
 --- | ---
 Enabled | No
 
-This cop checks for duplicate gem entries in Gemfiles. Bundler currently
-only prints a warning (unless there is a requirements conflict).
+A Gem's requirements should be listed only once in a Gemfile.
```

Comment:
This seems to happen more or less every time. Can we somehow add some programmatic enforcement for this, similar to what we do for the `CHANGELOG`? 😀